editor/src/components/inspector/sections/component-section/component-section.tsx

```diff
@@ -1164,54 +1187,89 @@ export class ComponentSection extends React.Component<
 
 interface VariableOption {
   variableName: string
+  displayName: string
   definedElsewhere: string | null
   value: string
+  depth?: number
 }
 
 function valuesFromObject(
   name: string,
   objectName: string,
   value: object | null,
+  depth: number = 0,
+  displayName: string = name,
```

Comment:
We should avoid using default parameters because they muddle up the function's API for a limited degree of convenience